.tools/test/sqs_lambda_to_batch_fargate/consumer_stack/consumer_stack.py

```diff
@@ -44,7 +44,8 @@ def get_yaml_config(self, filepath):
         return data
 
     def init_get_topic(self, topic_name):
-        topic = sns.Topic(self, "fanout-topic", topic_name=topic_name)
+        external_sns_topic_arn = f"arn:aws:sns:us-east-1:{self.producer_account_id}:{topic_name}"
```

Comment:
Will hardcoding the region cause problems?

Reply:
Not yet, but maybe down the road. Added it as a config value. Good call out.